core/src/test/java/brooklyn/catalog/internal/CatalogScanTest.java

```diff
@@ -161,13 +162,14 @@ public void testMoreTypesThanAnnotationsForApps() {
     
     @Test
     public void testAnnotationIsDefault() {
```

Comment:
This test started failing for me after the merge of #634 
Seems to be caused by me having a default catalog in ~/.brooklyn which replaces the `default.catalog.bom` expected here.